game-service/src/main/java/org/libertybikes/game/round/service/GameRoundWebsocket.java

```diff
@@ -77,7 +77,9 @@ else if (!round.isPlayer(session))
             if (GameEvent.GAME_START == msg.event) {
                 round.startGame();
             } else if (msg.direction != null) {
-                round.updatePlayerDirection(session, msg);
+                if (round.gameState == State.RUNNING) {
```

Comment:
so I know this was the one thing we were originally planning to do in this PR, but lets not make this change in favor of what @realModusOperandi suggested for having sprites rotate properly during the countdown. 
The rest of the changes in this PR are good though and should be delivered

Reply:
Got it working